test/integration/apiserver/tracing/tracing_test.go

```diff
@@ -513,7 +513,7 @@ endpoint: %s`, listener.Addr().String())), os.FileMode(0755)); err != nil {
 		{
 			desc: "list nodes",
 			apiCall: func(c *client.Clientset) error {
-				_, err = clientSet.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
+				_, err = clientSet.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{Limit: 1})
```

Comment:
This will no longer force the request to go to etcd as soon as https://github.com/kubernetes/kubernetes/pull/125405 merges.

What's the purpose of this change?